packages/app/src/utils/API/fetchNomination.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios'
+
+const fetchNomination = (id) => {
+    axios.get(`/nomination/${id}`)
+        .then(function (response) {
+            console.log(response)
+            const nomination = response.data.nomination
+            return nomination
+        })
+        .catch(function (err) {
+            console.log(err)
+        })
+        .then(function() {
+        })
```

Comment:
The idea of breaking out the API call makes sense, but maybe you want to handle the actual response in the component (which you did by moving the axios call there, but potentially these approaches could be combined)?